tests/test_project_creation.py

```diff
@@ -105,6 +105,10 @@ def test_project_details(self):
         button = driver.find_element(By.ID, 'submit-button')
         driver.execute_script("window.scrollTo(0, document.body.scrollHeight);")
         self.wait.until(EC.element_to_be_clickable(button)).click()
+        
+        assert driver.current_url == "http://localhost:8000/dashboard"
+        assert self.project_name in driver.find_element(By.ID, "my-projects").text
```

Comment:
Can we also validate the data displayed on the summary screen is as expected?  This is doing a fetch of the data from the API so is useful to validate that we don't have issues that side of the fence too.

Reply:
> Can we also validate the data displayed on the summary screen is as expected? This is doing a fetch of the data from the API so is useful to validate that we don't have issues that side of the fence too.

Fixed